plugin/mutate/README_roadmap.md

```diff
@@ -0,0 +1,98 @@
+# Roadmap
+
+The development of dextool mutate is focused on implementing a mutation testing
+tool which just works. The use cases can be found [here](#req-use_cases).
+
+The vision that the design should strive for is an architecture that enables
+functional components to freely added and replaced.
+
+Assume that the future is uncertain.
+
+Architectural goals:
+ * it should be *expandable* with new modules without affecting existing modules
+ * it should be possible to parallelize
+ * the programming language a module is written in should be left as an
+   implementation detail for the module
+ * the architecture should not limit the programing languages that can be
+   mutation tested. It should be possible to use the architecture for mutation
+   testing of any programming language
+ * enable reuse of components. A visualization component should be reusable
+   independent of the programming language that is mutation tested
+ * it should be possible to incrementally use mutation testing during development.
+ * it should be robust to infrastructure failures
+
+The purpose of this plugin is to perform mutation testing of a _target_.
+
+The focus of this plugin is to use mutation testing to automate whole or part
+of the manual inspection of test procedures that are required when developing
+software to DAL-A and C according to RTCA-178C.
```

Comment:
Suggestion: I think this should be removed.

Reply:
it is part of the purpose but it should be reformulated because it is generic